0dyseus@PopupTranslator/files/0dyseus@PopupTranslator/metadata.json

```diff
@@ -1,17 +1,18 @@
 {
-    "dangerous": false,
-    "uuid": "0dyseus@PopupTranslator",
-    "max-instances": "-1",
-    "description": "Simple translator applet that will allow to display any selected text from any application on a system in a popup.",
-    "multiversion": true,
-    "version": "1.02",
-    "name": "Popup Translator",
-    "comments": "Bug reports and feature requests should be reported on this applet's repository linked below.",
-    "website": "https://github.com/Odyseus/CinnamonTools",
-    "contributors": "NikoKrause: German localization and bug reports.,Radek71: Czech localization and bug reports.",
+    "website": "https://github.com/Odyseus/CinnamonTools", 
+    "multiversion": true, 
+    "last-edited": 1486145103, 
```

Comment:
Okay. Let's try it. Only hesitated because I already broken the update mechanism once a few days ago^^

The reorganization of the metadata.json file happens to me as well. It always puts the dangerous flag in there. I don't want it for my applet, so I `git checkout metadata.json` before commit.